installer/build-script.sh

```diff
@@ -72,6 +77,15 @@ fi
 ${INSTALLER_CMD}
 EOF
 
+# dkms may not be available as rpm - in that case install it here
+if [ ! command -v dkms &> /dev/null ]
```

Comment:
This is outside os specific check and assumes yum availability. It will work for now as dkms is missing only on amazon linux.

Reply:
seem to be missing on latest Amazon Linux 2, but you are correct and I will move it under that condition above, where we check for yum first